src/Pages/Register.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, React } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import AuthForm from "../components/AuthForm/AuthForm";
+import { useNavigate } from "react-router-dom";
+import {
+  registeredUser,
+  resetRegisterState,
+} from "../rtk/features/register/registerSlice";
+import Layout from "../components/Layout/Layout";
+
+const Register = () => {
+  const { success } = useSelector((state) => state.register);
+
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const handleSubmit = (user) => {
+    user.id = Date.now();
```

Comment:
use [uuid](https://www.npmjs.com/package/uuidv4) instead of Date as an user id, this is not a good practice.